### DISCUSSION TITLE
How to keep index when using dask.dataframe.DataFrame.melt

### DISCUSSION THREAD

**Discussion by novdanody** [dask/dask#9013] [Q&A]
In latest pandas version, looks like there is an ignore_index argument that can be set to False to keep index, is there any way to melt while keeping index in Dask?

**Comment by novdanody** [comment]
```python
import numpy as np
import pandas as pd
import dask.dataframe as dd

col = pd.MultiIndex.from_product([['foo', 'bar'], ['a','b','c']])
df = pd.DataFrame(np.random.rand(5,6))
df.columns = col
df.index = ["k", "l", "m", "n", "o"]
print(df.melt(ignore_index=False)) # Want this

dask_df = dd.from_pandas(df, npartitions=1)
print(dask_df.melt().compute())

print(dask_df.melt(ignore_index=False).compute()) # this will throw saying ignore_index does not exist
```

**Reply by jrbourbeau** [reply]
Indeed, you're correct @novdanody it looks like nobody has implemented the `ignore_index` keyword for Dask's version of `melt`. IS this something you're interested in contributing? (no obligation though)

**Reply by novdanody** [reply]
For sure, would be happy to take a look (although I have not actively contributed to any open source project in the past). Do you have any suggestion/resources on best practices and how I can get started?
